platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -47,6 +47,48 @@ Note that certain features that rely on third-party tools, such as [PMD](https:/
 
 For details, see the full [compatibility documentation](userguide/compatibility.html#java).
 
+<a name="java-compiler-error-rendering"></a>
+### Java compiler errors in the failure report
+
+In previous Gradle versions, finding the reason of a failed compilation was a suboptimal experience: the only pointer we could give to the user was to scroll back, and look for the failed task's output, containing the compiler failure.
+
+This user experience wasn't great; build logs can be extremely long, and our support for multiple failures (when using the `--continue` flag) could complicate the situation further, making the identification of the exact failure reason challenging.
```

Comment:
```suggestion
This process was particularly challenging when using the `--continue` flag, as handling multiple failures could make it difficult to pinpoint the exact cause of an error.
```

Reply:
I think we lost some meaning there:
 - The key point is that this feature helps with long logs
 - I've tried to convey that if for example you have long logs, and use `--continue` (tbh, not a super commonly used feature), you might have to spend a decent time on finding the root issue of your compilation